### DISCUSSION TITLE
Who's maintaining this repo?

### DISCUSSION THREAD

**Discussion by amoorthaemer** [dotnet-state-machine/stateless#609] [General]
Hi all,

I've seem not much activitiy on this repo during the past time. I can see a lot of PR's, but nobody takes care of them.
Also missing .NET 9 as a target framework.

So, question: is this a dead repo? I'd like to use it and saw some good PR's (besides not having .NET 9) coming in, but as long nobody actually maintains this, this repo is a dead end for me

**Comment by adanbrownpaca** [comment]
I followed this repo and used it in production. While only a few are working on it, it is active and being monitored. Yes, I think the .Net 9 target should be added!

**Comment by mclift** [comment]
I joined as a maintainer a couple of years ago after another GitHub member raised a similar concern (see #466), although my activity here tends to come in fits and starts. I usually spend a bit of time on it around this time of year.

@nblumhardt 's first commit to this repo was in 2009 so it's understandable people have moved on. Is it time for a call for new maintainers?
